src/interpolation/interpolate.ts

```diff
@@ -66,8 +67,40 @@ const buildUpdates = (placeholders: Placeholder[]): Request[] => {
     // ContentPlaceholder
     if (placeholder.type == 'content') {
       const contentPlaceholder = placeholder as ContentPlaceholder
+      // Image
+      if (contentPlaceholder.output.url) {
```

Comment:
If we are waiting for an url here, we can't apply formatters on images. Big call, I know, but it would be nice to be able to crop the image or apply a sepia filter etc.

Reply:
Currently, we can only use urls. If we find a way to use a way to directly handle the image, then we will change this